releases/__init__.py

```diff
@@ -629,6 +629,12 @@ def setup(app):
         app.add_config_value(
             name='releases_{0}'.format(key), default=default, rebuild='html'
         )
+    # if a string is given for `document_name`, convert it to a list
+    # done to maintain backwards compatibility
+    if isinstance(app.config.releases_document_name, str):
```

Comment:
Probably best to `s/str/six.string_types` here in case anyone's on Python 2 + using Unicode string objects.